tests/functional/test_install.py

```diff
@@ -498,6 +499,41 @@ def test_hashed_install_failure(script, tmpdir):
     assert len(result.files_created) == 0
 
 
+def assert_re_match(pattern, text):
+    assert re.search(pattern, text), (
+        "Could not find {!r} in {!r}".format(pattern, text)
+    )
```

Comment:
FYI, https://github.com/asottile/re-assert could be useful, since it has pretty assertion error messages.

Reply:
It would be nice to see exactly where it failed to match! Feel free to submit a PR. :)

Reply:
Aw, it's Python 3 only :) Better to wait till drop the Python 2 support.